src/AFQMC/Estimators/tests/CMakeLists.txt

```diff
@@ -33,5 +33,5 @@ FOREACH(I RANGE 0 ${ENDP} 2)
     SET(UTEST_NAME deterministic-unit_test_${SRC_DIR}_${HAMIL}_${WFN})
     ADD_UNIT_TEST(${UTEST_NAME} "${QMCPACK_UNIT_TEST_DIR}/${UTEST_EXE}"
                   "--hamil ${HAMIL_FILE}" "--wfn ${WFN_FILE}")
-    SET_TESTS_PROPERTIES(${UTEST_NAME} PROPERTIES LABELS "unit;afqmc" WORKING_DIRECTORY ${UTEST_DIR})
+    SET_TESTS_PROPERTIES(${UTEST_NAME} PROPERTIES LABELS "unit;afqmc;deterministic" WORKING_DIRECTORY ${UTEST_DIR})
```

Comment:
Please use the following lines.
```
SET_TESTS_PROPERTIES(${UTEST_NAME} PROPERTIES WORKING_DIRECTORY ${UTEST_DIR})
SET_PROPERTY(TEST ${UTEST_NAME} APPEND PROPERTY LABELS "afqmc")
```
ADD_UNIT_TEST() adds unit and deterministic. 'afqmc' needs to be appended. The old code went wrong because of overwriting labels.

Reply:
Updated. Checked with cpu complex & gpu complex builds and ctest -L deterministic